### PR TITLE
Add travis.yml

### PR DESCRIPTION
**Pull request by dwalintukan**

### CODE REVIEW

File: test/contract.js [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
 import ContractMetadata from './data/contract_metadata';
 import Contract from '../src/contract';
 import Encoder from '../src/encoder';
+import Formatter from '../src/formatter';
 
 describe('Contract', () => {
   let contract;
 
   describe('constructor', () => {
-    it('inits all the values', async () => {
```
Comment (chrisli30) [inline test/contract.js]: yep, async not needed

File: test/contract.js [hunk 2]
```diff
@@ -21,38 +23,56 @@ describe('Contract', () => {
...
-    it('removes the hex prefix from the address', async () => {
+    it('removes the hex prefix from the address', () => {
       contract = new Contract(Config.QTUM_RPC_ADDRESS, '0x1234567890', ContractMetadata.EventFactory.abi);
       assert.equal(contract.address, '1234567890');
     });
   });
 
   describe('call()', () => {
-    it('returns the values', async () => {
+    it('returns the values', () => {
```
Comment (chrisli30) [inline test/contract.js]: This 'returns the values' name is blur to me. "Returns correct BN from <what result>"?

File: test/contract.js [hunk 3]
```diff
@@ -21,38 +23,56 @@ describe('Contract', () => {
...
       );
-
-      const res = await contract.call('getEventName', {
-        methodArgs: [],
-        senderAddress: Config.SENDER_ADDRESS,
-      });
-      assert.equal(res[0].replace(/\0/g, ''), 'Who will win the 2018 NBA Finals Championships?');
+      const formatted = Formatter.callOutput(result, ContractMetadata.CentralizedOracle.abi, 'bettingEndBlock', true);
+      assert.isTrue(Web3Utils.isBN(formatted['0']));
+      assert.equal(formatted['0'].toJSON(), '100e0');
```
Comment (chrisli30) [inline test/contract.js]: Ideally this "100e0" should not be hardcoded. The expected output and input should be generated from the same source in test.

For example,
in the beginning of test.
var input = "0x00000000100e0";
var output = convertToString(input);

in the end
assertEqual with output.

Comment (chrisli30) [reply test/contract.js]: I've approved this PR. If you decide to make changes, make sure test still passes and merge.

**Comment by dwalintukan** [comment]
Using the other Travis CI PR. This one is without the testnet block data included.

**Pull request closed without merging**